src/app/admin/createCollectionCentre/CreateCollectionCentreForm.tsx

```diff
@@ -102,8 +104,10 @@ const CreateCollectionCentreForm: React.FC<{}> = () => {
                     />
                 ))}
 
-                {refreshRequired ? (
-                    <RefreshPageButton />
+                {addCollectionCentreIsSuccess ? (
+                    <Alert severity="success" action={<RefreshPageButton />}>
+                        Collection centre <b>{fields.name}</b> added successfully.
```

Comment:
Does this mean if we successfully add a collection centre and then change the value of a field, does that changes the success message too?

Reply:
Good spot 🙈